Content.Server/Power/Components/BatterySelfRechargerComponent.cs

```diff
@@ -9,5 +9,11 @@ public sealed partial class BatterySelfRechargerComponent : Component
         [ViewVariables(VVAccess.ReadWrite)] [DataField("autoRecharge")] public bool AutoRecharge { get; set; }
 
         [ViewVariables(VVAccess.ReadWrite)] [DataField("autoRechargeRate")] public float AutoRechargeRate { get; set; }
+
+        [ViewVariables(VVAccess.ReadWrite)] [DataField("autoRechargePause")] public bool AutoRechargePause { get; set; } = false;
+
+        [ViewVariables(VVAccess.ReadWrite)] [DataField("autoRechargePauseTime")] public float AutoRechargePauseTime { get; set; } = 0f;
+
+        [ViewVariables(VVAccess.ReadWrite)] [DataField("autoRechargeCooldown")] public float AutoRechargeCooldown { get; set; } = 0f;
```

Comment:
I recall being told that ViewVariables is no longer needed as DataField has that functionality. But I might be wrong.

Reply:
Copying what was done in that component originally to be safe.